.github/workflows/build-and-deploy.yml

```diff
@@ -154,6 +170,16 @@ jobs:
         run: |
           cp -avr wiki-tutorials/target/generated-docs/ target/generated-docs/website/pages/learning/
 
+      - name: Create output for journeys
+        run: |
+          chmod +x journeys/scripts/createOutput.sh
+          journeys/scripts/createOutput.sh
```

Comment:
Please align with the rest of the team on the output path. Currently it looks strange